example/scripts/generate_ingest/generate_files.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable node/no-extraneous-require */
+/* eslint-disable no-console */
+
+const { s3PutObject } = require('@cumulus/aws-client/S3');
+const { randomStringFromRegex } = require('@cumulus/common/test-utils');
+
+const granuleRegExp = 'MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}';
+const BATCHSIZE = 100;
+
+// In batches of 100, upload granules
+const uploadTestFiles = async (Bucket, path, batches) => {
+  console.log(`Bucket is ${Bucket}`);
+  console.log(...process.argv.slice(2, 5));
+  let batchCount = 0;
+  console.log(`Running ${batches} batches`);
+  while (batchCount < batches) {
+    batchCount += 1;
+    console.log(`Uploading batch ${batchCount}`);
+    const uploadBatchPromises = new Array(BATCHSIZE).fill().map(
+      () => {
+        const granString = randomStringFromRegex(granuleRegExp);
+        return Promise.all(['hdf', 'jpg', 'hdf.met'].map((extension) => s3PutObject({
+          Bucket,
+          Key: `${path}/${granString}.${extension}`,
+          Body: 'fake_test_data',
+        })));
+      }
+    );
+    // eslint-disable-next-line no-await-in-loop
+    await Promise.all(uploadBatchPromises);
+    console.log(`Uploaded ${batchCount * BATCHSIZE} / ${BATCHSIZE * batches} granules`);
+  }
+};
+
+uploadTestFiles(...process.argv.slice(2, 5)).then(
```

Comment:
Suggestion: Would it be good to do a little bit of argument validation?

Reply:
Probably.  If the bucket isn't available, it's going to fail, and the same with a garbage path.    It might be worth bounding the number of batches to avoid inadvertently writing billions of keys. 

How about : 68c1bda75